jibri/rootfs/etc/cont-init.d/10-config

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/with-contenv bash
+export JIBRI_INSTANCE_ID=$(date +%N)
+
+# set external IP as xmpp domain
+echo "${DOCKER_HOST_ADDRESS}	${XMPP_DOMAIN}" >> /etc/hosts
```

Comment:
why is this needed?

Reply:
`JIBRI_INSTANCE_ID` set "uniq" id for the control_muc -> nickname.
/etc/hosts - I use it for the second case, but maybe will better use external `LETSENCRYPT_DOMAIN` in jibri/rootfs/defaults/config.json->xmpp_environments->xmpp_domain instead. But again, its not clear

Reply:
I don't understand what you mean. But since you mentioned the second case, let's drop it please.